rust/src/server.rs

```diff
@@ -91,7 +113,14 @@ where
                 None => return Ok(Status::unauthenticated("missing authorization").to_http()),
             };
 
-            info!("Authenticated request: {}", req.uri());
+            let tag_str = match tag.to_str() {
+                Ok(tag) => tag,
```

Comment:
Why both "tag" and "token"?

Reply:
token is the random string of characters used as a token, tag is a name associated with that token.

Reply:
Can you add that as a comment somewhere in the code? It's not clear that that's the case to me.

Reply:
I'm going to update it to username since it's more traditional than "tag". The original plan was to allow for multiple tags per token, but I don't currently have a need for that.